Add 3 more Weather icons

bach5000 opened the pull request:
For issue #8:
Added new icons for 3 weather conditions:
* hurricane
* scattered snow showers
* snow showers

bach5000 commented:
So far they look like this:
Hurricane:
![weather_hurricane](https://user-images.githubusercontent.com/17076625/48065201-ae8f4d00-e18f-11e8-972a-98ad0da89d87.png)
Scattered snow showers:
![weather_scattered_snow_showers](https://user-images.githubusercontent.com/17076625/48065202-ae8f4d00-e18f-11e8-91c8-9bb924d730f5.png)
Snow showers:
![weather_snow_showers](https://user-images.githubusercontent.com/17076625/48065203-ae8f4d00-e18f-11e8-91aa-56aa6241a801.png)

humdingerb commented:
Looking good.

The cloud shadows of the 'Scattered snow showers' isn't quits right. They should be a bit smaller and nearer to the clouds. The snow flakes should be fewer than for 'Snow showers' and should be distributed to use the whole height of the icon (i.e. from cloud to bottom of the icon).

I wonder if using a gradient for the 'Hurricane' that is green with only a small portion white (or very light green) could be used to make the waves look like they got some froth at the tips...

bach5000 commented:
@humdingerb Sure :)

bach5000 commented:
@humdingerb: Changed some icons :)
* changed shadow size and placement
* added a little gradient to the water

Hurricane
![weather_hurricane](https://user-images.githubusercontent.com/17076625/48102391-6ce2be80-e1f0-11e8-9669-4b5eda41e737.png)

Scattered Snow Showers
![weather_scattered_snow_showers](https://user-images.githubusercontent.com/17076625/48102410-7c620780-e1f0-11e8-84e1-caa2f5465422.png)
